recursive_iterativeOddSum.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+const recursiveOddSumTo = (number) => {

```

Comment:
please, fix `recursiveOddSumTo(0)` variant

Reply:
Fixed
Tried to make it better than simple `if`, but no.